core/templates/core/organismAnnotation.html

```diff
@@ -48,23 +48,37 @@
                                     <div class="row justify-content-center p-4">
                                         <div class="col-md-9">
                                             <div class="card ">
-                                                <div class="card-header text-center"><h2 style="text-align:center">Load gff anotation file</h2> </div>
+                                                <div class="card-header text-center"><h2 style="text-align:center">Load GFF anotation file</h2> </div>
                                                 <form method="post" enctype="multipart/form-data" name="uploadAnnotation"  id="uploadAnnotation" class="form-horizontal well" style="min-height: 100px;">
                                                     <div class="card-body">
                                                         {% csrf_token %}
                                                         <input type="hidden" name="action" value="uploadAnnotation">
-                                                        <div class="col-md-9">
-                                                            <label for="schemafile" class="form-label">Select gff file</label>
-                                                            <input class="form-control" type="file" required accept=".gff, .gf3" id="gffFile", name="gffFile">
+                                                        <div class="col-md-12">
+                                                        <label for="schemaFile" class="form-label">Select GFF File</label>
+                                                        <div class="input-group">
+                                                            <input type="file" class="form-control" id="schemaFile" name="schemaFile" required hidden onchange="updateFileName(this)">
+                                                            <button type="button" class="btn btn-outline-secondary" onclick="document.getElementById('schemaFile').click();">Choose File</button>
```

Comment:
```suggestion
                                                        <label for="gffFile" class="form-label">Select GFF File</label>
                                                        <div class="input-group">
                                                            <input type="file" class="form-control" id="gffFile" name="gffFile" required hidden accept=".gff, .gff3" onchange="updateFileName(this)">
                                                            <button type="button" class="btn btn-outline-secondary" onclick="document.getElementById('schemaFile').click();">Choose File</button>
```
I think you copied the same code twice